packager/app/packager_main.cc

```diff
@@ -106,7 +106,10 @@ const char kUsage[] =
     "    The group ID for the output stream. For HLS this is used as the\n"
     "    GROUP-ID attribute for EXT-X-MEDIA.\n"
     "  - playlist_name: Required for HLS output.\n"
-    "    Name of the playlist for the stream. Usually ends with '.m3u8'.\n";
+    "    Name of the playlist for the stream. Usually ends with '.m3u8'.\n"
+    "  - skip_encryption: Optional value which specifies if encryption of\n"
+    "    the stream shall be skipped. By default, if encryption is enabled,\n"
```

Comment:
Make it clear that it expects a numeric value instead of "true|false", e.g.

skip_encryption: Optional, if set to nonzero, encryption of the stream shall be skipped. ...